build/KoreBuild.ps1

```diff
@@ -42,8 +54,9 @@ if ($env:KOREBUILD_SKIP_RUNTIME_INSTALL -eq "1")
 }
 else
 {
+    # Install the version of dotnet-cli used to compile
     & "$koreBuildFolder\dotnet\dotnet-install.ps1" -Channel $dotnetChannel -Version $dotnetVersion -Architecture x64
-    InstallSharedRuntime '1.0.0' 'preview'
```

Comment:
A few of our tools do, but the 1.0.0-rc4 SDK installs both the 1.0.4 and 1.1.1 runtimes automatically